src/algebra/group/defs.lean

```diff
@@ -611,6 +820,22 @@ class add_comm_group (G : Type u) extends add_group G, add_comm_monoid G
 attribute [to_additive] comm_group
 attribute [instance, priority 300] add_comm_group.to_add_comm_monoid
 
+-- TODO: this is super slow for some reason!
```

Comment:
This proof is really slow compared to the others like this. Is there a faster way to get this?

Reply:
This is less than a second, whereas your proof is about 15 seconds on my machine.
```
begin
  rintros ⟨⟩ ⟨⟩ h,
  replace h := group.mk.inj h,
  dsimp at h,
  rcases h with ⟨rfl, rfl, rfl, rfl, rfl, rfl⟩,
  refl
end
```